src/vis/bar/BarVis.tsx

```diff
@@ -65,6 +67,9 @@ export function BarVis({
   columns,
   setConfig,
   scales,
+  selectionCallback = () => null,
+  selectedMap = {},
+  selectedList = [],
```

Comment:
More or less yes, selectedList is what gets passed into the vis component and we internally create selectedMap to help with speed stuff. We pass them both around frequently since they are used in different ways, but we could just pass around selectedMap and infer the list as you said. But this seems slightly more confusing to me than just using the list that we already have?

Added types for the map

Reply:
Ok, then we leave it like this for now.